tanzu-java-web-app/accelerator.yaml

```diff
@@ -61,8 +59,10 @@ engine:
   let:
     - name: updateBoot3
       expression: "#nativeBuild == true ? true : ( #springBootVersion == '2.7' ? false : true )"
+    - name: nativeJavaVersion
+      expression: "#javaVersion == '17' ? '17' : ( #javaVersion == '21' ? '21' : '17' )"
     - name: javaVersion
```

Comment:
I'm assuming these update the build pack environment variable in the workload.yaml to use the appropriate java version. Is this correct?

Reply:
Yes, that's what they do, if they select boot 3.2, required for native we override javaVersion with 17 or 21 depending on what they selected